### PR TITLE
removes unused pressure arg from Irradiance.liujordan function

### PR DESCRIPTION
**Pull request by KonstantinTr**
- [x] Closes issue #386 
 - [x] Fully tested. Added and/or modified tests to ensure correct behavior for all reasonable inputs. Tests must pass on the TravisCI and Appveyor testing services.
 - [x] Code quality and style is sufficient. Passes ``git diff upstream/master -u -- "*.py" | flake8 --diff`` and/or landscape.io linting service.
 - [x] New code is fully documented. Includes sphinx/numpydoc compliant docstrings and comments in the code where necessary.
 - [x] Updates entries to `docs/sphinx/source/api.rst` for API changes.
 - [x] Adds description and name entries in the appropriate `docs/sphinx/source/whatsnew` file for all changes.

### CODE REVIEW

File: docs/sphinx/source/whatsnew/v0.5.2.rst [hunk 1]
```diff
@@ -14,6 +14,7 @@ Enhancements
 Bug fixes
 ~~~~~~~~~
 * fixed redeclaration of test_simplified_solis_series_elevation (:issue:`387`)
+* removed unused 'pressure' arg from irradiance.liujordan function (:issue:`386`)
```
Comment (wholmgren) [inline docs/sphinx/source/whatsnew/v0.5.2.rst]: I suggest putting this in the API changes section to give it a little more visibility.

File: pvlib/test/test_irradiance.py [hunk 2]
```diff
@@ -201,8 +201,7 @@ def test_liujordan():
         columns=['ghi', 'dni', 'dhi'],
         index=[0])
     out = irradiance.liujordan(
-        pd.Series([10]), pd.Series([0.5]), pd.Series([1.1]),
-        pressure=93000., dni_extra=1400)
+        pd.Series([10]), pd.Series([0.5]), pd.Series([1.1]),dni_extra=1400)
```
Comment (wholmgren) [inline pvlib/test/test_irradiance.py]: space after comma

**Comment by KonstantinTr** [comment]
I don't like that tests fail on python 3.

When I run on my local windows machine I've got: 457 total, 1 failed, 5 ignored, 451 passed

**Comment by KonstantinTr** [comment]
INSTALLED VERSIONS
------------------
commit: None
python: 3.6.3.final.0
python-bits: 64
OS: Windows
OS-release: 10
machine: AMD64
processor: Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
byteorder: little
LC_ALL: None
LANG: None
LOCALE: None.None

pandas: 0.20.3
pytest: 3.2.1
pip: None
setuptools: 36.5.0.post20170921
Cython: 0.26.1
numpy: 1.13.3
scipy: 0.19.1
xarray: None
IPython: 6.1.0
sphinx: 1.6.3
patsy: 0.4.1
dateutil: 2.6.1
pytz: 2017.2
blosc: None
bottleneck: 1.2.1
tables: 3.4.2
numexpr: 2.6.2
feather: None
matplotlib: 2.1.0
openpyxl: 2.4.8
xlrd: 1.1.0
xlwt: 1.3.0
xlsxwriter: 1.0.2
lxml: 4.1.0
bs4: 4.6.0
html5lib: 0.9999999
sqlalchemy: 1.1.13
pymysql: None
psycopg2: None
jinja2: 2.9.6
s3fs: None
pandas_gbq: None
pandas_datareader: None

**Comment by wholmgren** [comment]
Looks like we've resolved all the testing issues, so I'm going to go and merge this. Thanks for the fix @KonstantinTr

**Pull request merged**